### PR TITLE
fix vfx stuck after talking with ally

### PR DESCRIPTION
**Pull request by yalania**
Issue was because I was doing a disable of all the player GO, so when enabling ti again all the billboards were enable by default. Now I'm only disable the component mesh renderer

### CODE REVIEW

File: Scripts/src/Script/AllyInteractionsScript.h [hunk 1]
```diff
@@ -95,8 +96,8 @@ class AllyInteractionsScript : public Script
 	bool current_lightball_speed = 0.005f;
 	float time_passed = 0.0f;
 	//Camera view
-	GameObject* player_1_mesh = nullptr;
-	GameObject* player_2_mesh = nullptr;
+	Component* player_1_mesh = nullptr;
```
Comment (raulgonzalezupc) [inline Scripts/src/Script/AllyInteractionsScript.h:99]: why not ComponentMeshRenderer?

**Review by raulgonzalezupc — APPROVED** [review]
Tested and seems that works

File: Scripts/src/Script/AllyInteractionsScript.h (hunk 1 above)
Comment (yalania) [reply Scripts/src/Script/AllyInteractionsScript.h:99]: because I don't need to know specific things of componentMeshRenderer and I can avoid a casting.

**Review by marcorod94 — COMMENTED** [review]
Looks nice but @yalania I don't if you noticed the same thing happens when you revive a player could you look into it?

**Comment by yalania** [comment]
oki

**Review by marcorod94 — APPROVED** [review]
Nice Job

**Pull request merged**